CHANGELOG.rst

```diff
@@ -25,10 +25,12 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Make adding a new affiliation more intuitive (`#467 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/467>`_) (`#463 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/463>`_)
 
-* Harmonized Title and label structure across all views(`#455 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/455>`_)
+* Harmonized Title and label structure across all views (`#455 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/455>`_)
 
 * Updating a person removes the old entry also from the customerResourceService and projectManagerResourceService (`#456 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/456>`_)
 
+* Make empty address addition explicit (`#474 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/474>`_)
```

Comment:
```suggestion
* Make empty address addition explicitly selectable during customer creation/update  (`#474 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/474>`_)
```

Reply:
```suggestion
* Make empty address addition explicitly selectable during person creation and update (`#474 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/474>`_)
```

Reply:
If you agree with my suggestion you can accept it :)

Reply:
done 👍